vagrant-post-script/orchestrate_docker.sh

```diff
@@ -1,11 +1,13 @@
 #!/bin/bash
 
 cd /vagrant/
-#git reset --hard HEAD
-#./helper.sh --bumpmodules
+
 if [[ -f .env ]]; then
+  dos2unix .env
   make start
 else
+  git reset --hard HEAD
```

Comment:
what's this line for?

Reply:
encoding change between dos and unix

Reply:
I mean `git reset` line

Reply:
by resetting, the wrong encoding disappears

Reply:
weird, but ok